OptimizelySDK/DatafileManagement/HttpProjectConfigManager.cs

```diff
@@ -0,0 +1,237 @@
+﻿/* 
+ * Copyright 2019, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using OptimizelySDK.Logger;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OptimizelySDK.DatafileManagement
+{
+    public class HttpProjectConfigManager : PollingProjectConfigManager
+    {
+        private string Url;
+        private string LastModifiedSince = string.Empty;
+
+        private HttpProjectConfigManager(TimeSpan period, string url, ILogger logger, TimeSpan blockingTimeout) : base(period, blockingTimeout, logger)
+        {
+            Url = url;
+        }
+
+        public Task OnReady()
+        {
+            return CompletableConfigManager.Task;
+            //Task t = new Task(() => {
+            //    CompletableConfigManager.Task.Wait(this.BlockingTimeout);
+            //});
+            //t.Start();
+            //return t;
+        }
+        static ProjectConfig ParseProjectConfig(string datafile)
```

Comment:
I don't think this is the right place to put it. If you look at the Java SDK, the notion of a ProjectConfig backed by a Datafile is split from this and is defined at a higher level: https://github.com/optimizely/java-sdk/blob/master/core-api/src/main/java/com/optimizely/ab/config/DatafileProjectConfig.java

The idea is that we can have other types of Datafile-backed project config that don't necessarily have to be polled, such as push-based with websockets or server-sent events.